tests/integration/let-test.js

```diff
@@ -6,6 +6,12 @@ import {
 } from 'ember-mocha';
 import hbs from 'htmlbars-inline-precompile';
 import emberVersionIs from 'ember-version-is';
+import Ember from 'ember';
+
+const {
+  assign,
+  create
+} = Object;
```

Comment:
`Object.assign` is not present in many browsers (including Phantom which we are testing on).  Should likely swap to `Ember.merge` for compat with all Ember versions.